Sources/HelperKit/Extensions/UIView.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import UIKit
```

Comment:
- We don't need the `Foundation` import as we're also importing `UIKit`.
- Please add a `#canImport` guard like the one on `UIViewController`.